docs/building-images.md

```diff
@@ -107,6 +115,11 @@ operatingSystem:
   * `disable` - Optional; List of systemd services to disable.
 * `keymap` - Optional; sets the virtual console (VC) keymap, full list via `localectl list-keymaps`. If unset, we default to
   `us`.
+* `packages` - Optional; Defines packages that should have their dependencies determined and pre-loaded into the built image. For detailed information on how to use this configuration, see the [Installing pacakges](installing-packages.md) guide.
+  * `noGPGCheck` - Optional; Defines whether GPG validation should be done over all additional repositories and side-loaded RPMs. If set to `true` it effectively disables all GPG validaiton. If unset, defaults to `false`. **Disabling GPG validation is intended for development purposes only!**
```

Comment:
Nit: Please consider the following wording. Ignore if you prefer more detailed explanation.
```suggestion
  * `noGPGCheck` - Optional; Defines whether GPG validation should be disabled for all additional repositories and side-loaded RPMs. **Disabling GPG validation is intended for development purposes only!**
```